api/routers/auth-router.js

```diff
@@ -0,0 +1,43 @@
+//* all authentication routes (via passport) live here - for the time being Google is being used at the only 'strategy'
+
+//* create an instance of an express router
+const express = require("express");
+const router = express.Router();
+const googleAuth = require("../../passport-google");
+
+router.get("/login", (request, response) => {
+	// response.render("login", { user: request.user });
+	console.log(response);
+	response.redirect("https://leitspeed-fe.vercel.app/login");
+});
+
+// GET route for when you click on login - passport authenticates through google
+router.get(
+	"/google",
+	googleAuth.passport.authenticate("google", {
+		scope: ["openid email profile"],
+	})
+);
+
+//* If successful auth - redirects to home page, if not - redirects to /login
+router.get(
```

Comment:
If I'm reading this right, all the Google login routes are going through the "/auth/" router endpoints? And the local Passport routes are server level. Can we combine and abstract these logins together?

Reply:
Yea totally I was just trying to merge them together with the least amount of changes for now. But yes I think that would be ideal and could even add the strategies in the same file so we don't have two separate passport configs